rpcs3/rpcs3qt/ipc_settings_dialog.cpp

```diff
@@ -0,0 +1,69 @@
+#include <QMessageBox>
+#include <QHBoxLayout>
+#include <QPushButton>
+#include <QGroupBox>
+#include <QCheckBox>
+#include <QLineEdit>
+
+#include "ipc_settings_dialog.h"
+#include "Emu/IPC_config.h"
+
+ipc_settings_dialog::ipc_settings_dialog(QWidget* parent)
+	: QDialog(parent)
+{
+	setWindowTitle(tr("IPC Settings"));
+	setObjectName("ipc_settings_dialog");
+	setMinimumSize(QSize(200, 100));
+
+	QVBoxLayout* vbox_global = new QVBoxLayout();
+
+	QCheckBox* checkbox_server_enabled = new QCheckBox(tr("Enable IPC Server"));
+
+	QGroupBox* group_server_port = new QGroupBox(tr("IPC Server Port"));
+	QHBoxLayout* hbox_group_port = new QHBoxLayout();
+	QLineEdit* line_edit_server_port = new QLineEdit();
+
+	QHBoxLayout* hbox_group_buttons = new QHBoxLayout();
+	QPushButton* btn_save = new QPushButton(tr("Save"), this);
+
+
+	hbox_group_port->addWidget(line_edit_server_port);
+	group_server_port->setLayout(hbox_group_port);
+	hbox_group_buttons->addWidget(btn_save);
+
+	vbox_global->addWidget(checkbox_server_enabled);
+	vbox_global->addWidget(group_server_port);
+	vbox_global->addLayout(hbox_group_buttons);
+
+	setLayout(vbox_global);
+
+	connect(btn_save, &QAbstractButton::clicked, this, [this, checkbox_server_enabled, line_edit_server_port]()
```

Comment:
```suggestion
	connect(buttons, &QDialogButtonBox::accepted, this, [this, checkbox_server_enabled, line_edit_server_port]()
```

Reply:
I think I got it right? Still needed the validation on the port number, the validator only makes sure it's an integer with an allowed number of characters.

Reply:
I personally would've used a spinbox.
but it's uncommon for a port, so I think lineedit is better suited.
you still have to check the data though since you never know what nonsense Qt might throw at us.

Reply:
The IntValidator makes sure you can only input integers, but I'll check ok anyway just to be sure.